rsc/extra/check_js_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
```

Comment:
I have two little suggestions. :D

First is to use:

`#!/bin/sh` instead of `#!/bin/bash`

The second is to replace:

`#!/bin/bash -e` by

```sh
#!/bin/bash

set -eu
```

So that, one can launch the script with e.g. `sh script.sh` and still get the `-e` behavior. Also with adding `-u`, the script will fail on uninitialized variables.